paper/main.tex

```diff
@@ -78,25 +101,19 @@
 \begin{document}
 \linenumbers
 
-\title{Implicit assumptions and their impact on photometric redshift PDF performance for LSST}
-% \title{Photometric redshift uncertainties for LSST \romannumeral{1}: A unified methodology for assessing photo-$z$ PDF performance}
+\title{Evaluation of probabilistic photometric redshift estimation approaches for \textsc{LSST}}
```

Comment:
I think we were sort of asked to be more LSST-specific though, no?

Reply:
I thought we were specifically asked to make it more DESC/LSST-relevant. . .

Reply:
It's been a while now, but I think there was some discussion of how it was less DESC releveant because it didn't do tomography and stacking (still don't really agree that it's not relevant), but it's certainly still relevant for LSST, so I think LSST in the title is fine.